torch/_inductor/compile_worker/__main__.py

```diff
@@ -32,8 +32,9 @@ def main():
         args = parser.parse_args()
         if os.getppid() != args.parent:
             sys.exit(0)
-        read_fd = os.fdopen(args.read_fd, "rb")
-        write_fd = os.fdopen(args.write_fd, "wb")
+        with _WaitCounter("pytorch.file_system_access").guard() as _:
+            read_fd = os.fdopen(args.read_fd, "rb")
```

Comment:
I think this isn't a file and this is just using file descriptors for interprocess communication.